go-selinux/selinux_stub.go

```diff
@@ -96,6 +96,14 @@ func SetExecLabel(label string) error {
 	return nil
 }
 
+/*
+SetSocketLabel sets the SELinux label that the kernel will use for any programs
+that are executed by the current process thread, or an error.
+*/
+func SetSocketLabel(label string) error {
+	return nil
+}
+
```

Comment:
Please just ignore if this is a stupid question: Is there no need here for `SocketLabel()`?